app/views/notifier/invite.markerb

```diff
@@ -1 +1,8 @@
-<%= t('.message', :invite_url => invite_code_url(@invitation_code), :diasporafoundation_url => 'https://diasporafoundation.org/', :diaspora_id => @inviter.try(:diaspora_handle)) %>
+<%= t('.message',
+  invite_url: invite_code_url(@invitation_code),
+  diasporafoundation_url: 'https://diasporafoundation.org/',
+  user: @inviter.try(:diaspora_handle) == @inviter.try(:name) ?
```

Comment:
Indeed, but that's a big refactor (291 results in 99 file for .name) and I don't want to do it in this PR. I'll create a new method which will return nil. How should I call it at the moment?